dags/copy_deduplicate.py

```diff
@@ -89,38 +89,6 @@
 
     # todo: remove
 
-    # Experiment enrollment aggregates chain
-
-    gen_query_task_id = "experiment_enrollment_aggregates_live_generate_query"
-
-    # setting xcom_push to True outputs this query to an xcom
-    experiment_enrollment_aggregates_live_generate_query = gke_command(
-        task_id=gen_query_task_id,
-        command=[
-            "python",
-            "sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_live/view.sql.py",
-            "--submission-date",
-            "{{ ds }}",
-            "--json-output",
-            "--wait-seconds",
-            "15",
-        ],
-        docker_image="mozilla/bigquery-etl:latest",
-        xcom_push=True,
-        owner="ssuh@mozilla.com",
-        email=["telemetry-alerts@mozilla.com", "ssuh@mozilla.com"])
-
-    experiment_enrollment_aggregates_live_run_query = bigquery_xcom_query(
-        task_id="experiment_enrollment_aggregates_live_run_query",
-        destination_table=None,
-        dataset_id="telemetry_derived",
-        xcom_task_id=gen_query_task_id,
-        owner="ssuh@mozilla.com",
-        email=["telemetry-alerts@mozilla.com", "ssuh@mozilla.com"])
-
-    experiment_enrollment_aggregates_live_generate_query >> experiment_enrollment_aggregates_live_run_query
-
-
```

Comment:
This can be safely deleted since https://github.com/mozilla/bigquery-etl/pull/1680/files#r558609259 replaces this view